### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by portellaa**
Created getting started section for how to include the framework into the project

TODO list with some items to be done recently

### CODE REVIEW

**Comment by p4checo** [comment]
Please rebase 🔧

File: AlicerceTests/ServiceLocatorTests.swift [hunk 1]
```diff
@@ -0,0 +1,695 @@
...
+            let rStringService: String = try serviceLocator.get()
+            let rDoubleService: Double = try serviceLocator.get()
+            let rIntService: Int = try serviceLocator.get()
+            
+            // Validate registered services
+            XCTAssertEqual(rStringService, testServiceString)
+            XCTAssertEqual(rDoubleService, testServiceDouble)
+            XCTAssertEqual(rIntService, testServiceInt)
+            
+        } catch let error {
```
Comment (jakewelton) [inline AlicerceTests/ServiceLocatorTests.swift]: I don't think you need to declare the error variable here, as its now added by default.

File: AlicerceTests/ServiceLocatorTests.swift [hunk 2]
```diff
@@ -0,0 +1,695 @@
...
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+class ServiceLocatorTests : XCTestCase {
+    
+    var serviceLocator = ServiceLocator()
```
Comment (jakewelton) [inline AlicerceTests/ServiceLocatorTests.swift]: This is only a minor thing, but with `setUp` running before each test, do you think we need to create an object here? Normally I opt for an implicitly unwrapped optional `var serviceLocator: ServiceLocator!`.

What do you think?

File: AlicerceTests/ServiceLocatorTests.swift (hunk 1 above)
Comment (portellaa) [inline AlicerceTests/ServiceLocatorTests.swift]: Thanks @jakewelton, didn't know that.

File: AlicerceTests/ServiceLocatorTests.swift (hunk 2 above)
Comment (portellaa) [inline AlicerceTests/ServiceLocatorTests.swift]: Sorry, i don't like implicit unwrapped so i avoid to use it all the time. but yeah, it makes sense.

**Comment by portellaa** [comment]
@jakewelton this files moved from `AlicerceTests/Utils/ServiceLocatorTests.swift`

Sorry we are moving too fast with this, but we are working 200% on this with the new project.

Great work mate 🍺

File: AlicerceTests/ServiceLocatorTests.swift (hunk 2 above)
Comment (jakewelton) [inline AlicerceTests/ServiceLocatorTests.swift]: Thats fair enough. I too normally hate implicitly unwrapped options, but sometimes make an exception for unit tests as they are not production code. But thats fair enough 👍

**Comment by portellaa** [comment]
Hi @jakewelton

I did change the tests to do what you suggested, it makes a lot Of sense.

Sorry to not wait at this stage, but we really need to move forward with this part.

:beers_mug:

**Comment by jakewelton** [comment]
@portellaa That's fine, don't worry 😄 👍 However, I can't see the unit tests in the diff anymore for this PR. Were they moved to another PR?

**Pull request merged**